format/README.md

```diff
@@ -0,0 +1,18 @@
+# GraphAr Format Specification
+
+This folder contains protocol definitions for the GraphAr format.
+
+## How to generate code
+
+### Prerequisites
+
+- [protoc](https://developers.google.com/protocol-buffers/docs/downloads)
+- [buf](https://buf.build/docs/installation)
+
+### Build
+
+```sh
+buf generate
```

Comment:
It may be little confusing because one needs to run this command from the root of the project, not from the `format` folder

Reply:
I have added some wording to indicate that the command should run it the root of repository.